hadoop-ozone/dist/src/main/compose/ozones3/test.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+COMPOSE_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+export COMPOSE_DIR
+
+# shellcheck source=/dev/null
+source "$COMPOSE_DIR/../testlib.sh"
+
+start_docker_env
+
+execute_robot_test scm basic/basic.robot
```

Comment:
Yes, it is. It is the most basic level check if the compose folder is sill usable. (basic test checks only the availability of the webui and do a freon test with 5*5*5 keys. Maybe we can decrease the numbers to 1*1*5. (1 vol, 1 bucket, 1 key). If we can upload 5 keys, it should be fine.

Reply:
Ok, after some thinking I am understand. It may not be required all the time if we have more advanced tests. For example if the test plan contains a longer freon run, the basic test can be removed. 

But it's fast and an additional safety level (we don't start any test if basic freon doesn't work), so not a big problem but we can improve it later.